backend/search/index_data_to_elastic.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I believe we can restructure that a bit. There is no reason for `search` to reside inside the `backend`, it should be a separate folder/docker service. 

Scripts like `index_elastic_setup` should be executed as a part of the docker image build.

`index_data_to_elastic.js` could also reside inside the running elastic service (could be invoked by using `docker exec`. Since docker-compose provides the network connection between elastic service and mysql, reading and updating data won't be a problem.

Reply:
Additionally, `search` service can provide basic REST endpoint (batch indexing for the first version, incremental indexing in the future).

Reply:
> Scripts like index_elastic_setup should be executed as a part of the docker image build.

this is tricky to implement. Elasticsearch docker image has no folder/file mapping to execute something after container and service is started

Reply:
Check [this SO answer](https://stackoverflow.com/questions/35526532/how-to-add-an-elasticsearch-index-during-docker-build) for some ideas. Let me and @Lexy2 know if you're stuck, we can help research the right approach.